app/src/main/res/values-ru/strings.xml

```diff
@@ -345,8 +345,9 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="add_project">Добавить проект</string>
     <string name="archive_completed_tasks">Архивировать завершенные задачи</string>
     <string name="sort_alphabetically">Сортировать по алфавиту</string>
-    <string name="create_new_file_or_folder">Создать новый файл или папку</string>
+    <string name="create_new_file_or_folder">Создать новый файл или директорию</string>
```

Comment:
Директория is a collegial form of governance and has nothing to do with file system.
https://ru.wikipedia.org/wiki/Директория

Reply:
@mymedia2
Translations are managed at Crowdin.
Please translate & make suggestions at
https://crowdin.com/project/markor/invite

Reply:
I see. I already voted there for some translations that I think are right. :wink:

Reply:
@mymedia2
whats your crowdin name? I'm gonna give you proofread permission, so you can approve too. this means you can change the accepted string instead of only upvoting.
This would save me lot of time, because I don't know when somebody upvotes somewhere :D

Reply:
Thank you for your confidence. My nickname at Crowdin is `mymedia`.